### DISCUSSION TITLE
The requested resource `/docs/api` was not found on this server. (localhost)

### DISCUSSION THREAD

**Discussion by maxxwv** [dedoc/scramble#503] [Q&A]
I found [issue 201](https://github.com/dedoc/scramble/issues/201), but it doesn't seem to apply in this situation. Running `php artisan route:list` shows the expected routes and there are no conflicts, but I'm getting a 404 every time I try to access the URL. Both `env('APP_ENV');` and `config('app.env');` return 'local', so I assume I should be able to see the built docs.

I've got a base route defined on '/' that redirects to the login page so my default routing is working, it's just returning a 404 on the defined Scramble routes.

Using Laravel 11.20.0 and Scramble 0.11.11.

**Comment by romalytvynenko** [comment]
@maxxwv I am pretty sure this has something to do with your web server config
